src/main/steps/c100-rebuild/miam/miam-attendance/content.ts

```diff
@@ -0,0 +1,69 @@
+import { YesOrNo } from '../../../../app/case/definition';
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { isFieldFilledIn } from '../../../../app/form/validation';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const en = () => ({
+  headingTitle: 'Have you attended a Mediation Information and Assessment Meeting (MIAM)?',
+  paragraph1: 'The MIAM must be about the same issue that is being dealt with in this application.',
+  one: 'Yes',
+  two: 'No',
+  errors: {
+    miamAttendance: {
+      required: 'Select yes if you have attended a Mediation Information and Assessment Meeting(MIAM)',
+    },
+  },
+});
+
+const cy = () => ({
+  headingTitle: 'Have you attended a Mediation Information and Assessment Meeting (MIAM)? - welsh  ',
+  paragraph1: 'The MIAM must be about the same issue that is being dealt with in this application. - welsh',
+  one: 'Yes - Welsh',
+  two: 'No - Welsh',
+  errors: {
+    miamAttendance: {
+      required: 'Select yes if you have attended a Mediation Information and Assessment Meeting(MIAM) - welsh',
+    },
+  },
+});
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    miamAttendance: {
```

Comment:
@niranjan-301 Can we please follow naming conventions as informed ?
Please rename to 'miam_attendance'